tests/Test.ADAL.NET.Integration/ManagedUsercredentialTests.cs

```diff
@@ -69,5 +72,212 @@ public void Initialize()
             Assert.AreEqual(TestConstants.DefaultDisplayableId, result.UserInfo.DisplayableId);
             Assert.AreEqual(TestConstants.DefaultUniqueId, result.UserInfo.UniqueId);
         }
+
+        [TestMethod]
+        [Description("Positive Test for AcquireToken with a token already in cache")]
+        public async Task AcquireTokenWithValidTokenInCachePositiveTestAsync()
+        {
+            HttpMessageHandlerFactory.AddMockHandler(new MockHttpMessageHandler()
```

Comment:
for valid token in cache, no network call is made to user realm. you should update all tests to ensure that all mocks were consumed.